packages/plugins/i18n/admin/src/pages/SettingsPage/tests/SettingsPage.test.js

```diff
@@ -81,27 +81,26 @@ jest.mock('react-intl', () => ({
 
 describe('i18n settings page', () => {
   beforeEach(() => {
-    request.mockImplementation(() =>
-      Promise.resolve([
-        {
-          id: 1,
-          name: 'French',
-          code: 'fr-FR',
-          isDefault: false,
-        },
-        {
-          id: 2,
-          name: 'English',
-          code: 'en-US',
-          isDefault: true,
-        },
-      ])
-    );
-
-    useRBAC.mockImplementation(() => ({
-      isLoading: false,
-      allowedActions: { canRead: true, canUpdate: true, canCreate: true, canDelete: true },
-    }));
+    // request.mockImplementation(() =>
+    //   Promise.resolve([
+    //     {
+    //       id: 1,
+    //       name: 'French',
+    //       code: 'fr-FR',
+    //       isDefault: false,
+    //     },
+    //     {
+    //       id: 2,
+    //       name: 'English',
+    //       code: 'en-US',
+    //       isDefault: true,
+    //     },
+    //   ])
+    // );
+    // useRBAC.mockImplementation(() => ({
+    //   isLoading: false,
+    //   allowedActions: { canRead: true, canUpdate: true, canCreate: true, canDelete: true },
+    // }));
```

Comment:
All the tests are `todo` so I can't change anything here to make sure it works so i've just commented the rest of the stuff to avoid `eslint` complaining 👍🏼